tiptapy/templates/paragraph.html

```diff
@@ -1,5 +1,5 @@
 {%- if node.content -%}
-  <p>
+  <p{{ extract_parent_tag_attrs(node) }}>
```

Comment:
Why not use `node.attrs.get("id")` directly?

Reply:
because it will be added in couple of files, wanna keep logic at one place